Userland/Libraries/LibWasm/WASI/Wasi.cpp

```diff
@@ -160,18 +160,18 @@ SDFlags SDFlags::read_from(Array<ReadonlyBytes, 1> const& bytes)
 void FDStat::serialize_into(Array<Bytes, 1> bytes) const
 {
     auto data = bytes[0];
-    ABI::serialize(fs_filetype, Array { data.slice(offsetof(FDStat, fs_filetype), sizeof(fs_filetype)) });
-    ABI::serialize(fs_flags, Array { data.slice(offsetof(FDStat, fs_flags), sizeof(fs_flags)) });
-    ABI::serialize(fs_rights_base, Array { data.slice(offsetof(FDStat, fs_rights_base), sizeof(fs_rights_base)) });
-    ABI::serialize(fs_rights_inheriting, Array { data.slice(offsetof(FDStat, fs_rights_inheriting), sizeof(fs_rights_inheriting)) });
+    ABI::serialize(fs_filetype, Array { data.slice(__builtin_offsetof(FDStat, fs_filetype), sizeof(fs_filetype)) });
```

Comment:
This one seems unnecessary. `<stddef.h>` is one include away.

Reply:
Done.